vital/klv/klv_data.cxx

```diff
@@ -54,11 +54,11 @@ namespace vital {
 klv_data
 ::klv_data(container_t const& raw_packet,
          std::size_t key_offset, std::size_t key_len,
-         std::size_t m_value_offset, std::size_t value_len)
+         std::size_t value_offset, std::size_t value_len)
```

Comment:
oh, dear, shame on someone for using an "m_" as a parameter! :slightly_smiling_face: